tests/unit/s2n_utils_test.c

```diff
@@ -14,31 +14,33 @@
  */
 
 #include "s2n_test.h"
-
-#include "utils/s2n_safety.h"
-#include "utils/s2n_blob.h"
 #include "testlib/s2n_testlib.h"
+#include "utils/s2n_blob.h"
+#include "utils/s2n_safety.h"
 
-#define test_stack_blob_success(test_name, macro_name, requested, max) int test_name() {\
-macro_name(test_name ## blob, requested, max); \
-POSIX_ENSURE_EQ(test_name ## blob.size, requested); \
-return 0; \
-}
+#define test_stack_blob_success(test_name, macro_name, requested, max) \
+    int test_name()                                                    \
+    {                                                                  \
+        macro_name(test_name##blob, requested, max);                   \
+        POSIX_ENSURE_EQ(test_name##blob.size, requested);              \
+        return 0;                                                      \
+    }
 
 test_stack_blob_success(success_equal, s2n_stack_blob, 10, 10)
 
+        test_stack_blob_success(success_equal_smaller, s2n_stack_blob, 10, 100)
```

Comment:
I think something happened to the indentation here...It doesn't look right...

Reply:
Just need to add `test_stack_blob_success` to the list of Statement Macros.